src/Chainweb/Pact/Types.hs

```diff
@@ -389,6 +389,7 @@ defaultPactServiceConfig = PactServiceConfig
       , _pactUnlimitedInitialRewind = False
       , _pactBlockGasLimit = defaultBlockGasLimit
       , _pactLogGas = False
+      , _pactModuleCacheLimit = 1024^(3::Int) -- 1GiB
```

Comment:
Why is this different than the default for the config? Maybe we should define a default limit somewhere and use here, config, and for `TxSimulator` below (since it generally wants to mimic "production"